src/exifutils/pillowworker.py

```diff
@@ -144,6 +144,16 @@ def get_aperture(self) -> str:
         aperture = self.get_tag_with_log(ExifTags.Base.FNumber)
         return f"f/{aperture}" if aperture else "Unknown Aperture"
 
+    def get_dof_in_35mm(self) -> str:
+        focal_length = self.get_tag_with_log(ExifTags.Base.FocalLength)
+        focal_length_35mm = self.get_tag_with_log(ExifTags.Base.FocalLengthIn35mmFilm)
+        aperture = self.get_tag_with_log(ExifTags.Base.FNumber)
+
+        if aperture and focal_length and focal_length_35mm:
+            dof_in_35mm = round(float(focal_length_35mm / focal_length * aperture), 1)
+
+        return f"f/{dof_in_35mm}" if dof_in_35mm else "Unknown Depth of Field in 35mm format"
```

Comment:
`dof_in_35mm` is not initialized if the previous if branch is not triggered.

Reply:
This is fixed in a new commit, thanks.